src/main/java/ua/cv/tim/service/impl/UserServiceImpl.java

```diff
@@ -148,14 +150,14 @@ public void addUser(UserDTO member) throws MessagingException {
 		player.setAlliance(allianceDao.getAllianceByName(member.getAlliance()));
 		user.setPlayer(player);
 		playerDao.add(player);
-		sendEmail(user, password);
+		sendEmail(user,"Your login is \"" + user.getLogin() + "\", and password: \""
```

Comment:
I think, the second parameter here is redundant. You can create this message in sendEmail() method